Setup/Installer.php

```diff
@@ -108,6 +109,13 @@ public function install()
             true
         );
 
+        try {
+            // call the instance id service to create the instance id
+            (new InstanceIdService($this->connection))->getInstanceId();
+        } catch (Exception $e) {
+            throw new InstallationException($e->getMessage());
```

Comment:
as this is not critical for PayPal to work, I would not interrupt the installation. if we could not create the stuff for the instance ID -> :man_shrugging: